util/misc.py

```diff
@@ -76,3 +76,31 @@ def median(l):
                 return l[half]
         else:
             raise Exception("empty list for median")
+
+# from http://biopython.org/wiki/Split_large_file
+def batch_iterator(iterator, batch_size) :
```

Comment:
How about the "grouper" function in https://docs.python.org/2/library/itertools.html#recipes